products/sonarqube.md

```diff
@@ -23,7 +23,7 @@ releases:
     eol: false
     lts: 2023-02-07
     latest: "9.9.0"
-    latestReleaseDate: 2023-02-06
+    latestReleaseDate: 2023-03-02
```

Comment:
Using [tags may be more accurate](https://github.com/SonarSource/sonarqube/tags) here than using the docker image.

Reply:
Done : #2615